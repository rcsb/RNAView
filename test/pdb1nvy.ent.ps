%!PS-Adobe-2.0
%%Title: A postscript generated from a RNAML file.
%%CreationDate: Fri Mar 14 11:56:52 2008
%%Orientation: Portrait
%%BoundingBox:    251   268   361   524

/Times-Bold findfont 12  scalefont setfont


/Times-Bold findfont 13  scalefont setfont    %%set font for (A U G C T)

%% stacking diagram widths for base-pairs 1 & 2
/W1 {1 setlinewidth} bind def
/W2 {1.5 setlinewidth} bind def
/W3 {2 setlinewidth} bind def
/W4 {3 setlinewidth} bind def

%% minor and major grooves filling color saturation
/MINOR_SAT 0.9 def
/MAJOR_SAT 0.1 def
/OTHER_SIDES {0 0 1 sethsbcolor fill} bind def

%% defineing geometry shapes
/NP {newpath} bind def
/CIRCLE {0 360 arc closepath} bind def                        %% circle 
/TRIANGLE {moveto lineto lineto closepath} bind def           %% triangle
/SQUARE {moveto lineto lineto lineto closepath} bind def      %% square
/LINE {moveto lineto stroke} bind def                         %% line
/DASHLINE { moveto lineto [2 4] 0 setdash stroke } bind def %% dashline
/DOTLINE  {1 0 1 setrgbcolor} bind def                        %% dotline
/Dw {1 setlinewidth} bind def
/FB {setgray fill} bind def
/R6 {moveto lineto lineto lineto lineto lineto closepath} bind def
/R9 {moveto lineto lineto lineto lineto lineto
     lineto lineto lineto closepath} bind def

%% line drawing colors for ACGITU & others
/Al {0.0000 1.00 1.00 sethsbcolor} bind def
/Cl {0.1667 1.00 1.00 sethsbcolor} bind def
/Gl {0.3333 1.00 1.00 sethsbcolor} bind def
/Il {0.3333 1.00 0.57 sethsbcolor} bind def
/Tl {0.6667 1.00 1.00 sethsbcolor} bind def
/Ul {0.5000 1.00 1.00 sethsbcolor} bind def
/Xl {0.0000 0.00 0.00 sethsbcolor} bind def
/XX {0.2000 0.20 0.20 sethsbcolor} bind def
/FILLBLACK {0.0000 0.00 0.00 sethsbcolor fill} bind def

%% minor groove filling colors for ACGITU & others
/Am {0.0000 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Cm {0.1667 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Gm {0.3333 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Im {0.3333 MINOR_SAT 0.57      sethsbcolor fill} bind def
/Tm {0.6667 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Um {0.5000 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Xm {0.0000 0.00      MAJOR_SAT sethsbcolor fill} bind def

%% major groove filling colors for ACGITU & others
/AM {0.0000 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/CM {0.1667 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/GM {0.3333 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/IM {0.3333 MAJOR_SAT 0.57      sethsbcolor fill} bind def
/TM {0.6667 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/UM {0.5000 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/XM {0.0000 0.00      MINOR_SAT sethsbcolor fill} bind def

%% define line width, line join style & cap style (1 means round)
1 setlinewidth 1 setlinejoin 1 setlinecap
   271   288 translate

/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
70.00 211.83  moveto (T) center show 
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 70.00 182.67  moveto (C) center show grestore
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 70.00 153.50  moveto (G) center show grestore
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 70.00 124.33  moveto (G) center show grestore
70.00 95.17  moveto (T) center show 
70.00 66.00  moveto (A) center show 
70.00 48.50  moveto (C) center show 
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 70.00 31.00  moveto (C) center show grestore
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 70.00 13.50  moveto (G) center show grestore
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 70.00 -4.00  moveto (A) center show grestore
0.00 -4.00  moveto (T) center show 
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 0.00 13.50  moveto (C) center show grestore
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 0.00 31.00  moveto (G) center show grestore
0.00 48.50  moveto (G) center show 
0.00 66.00  moveto (T) center show 
0.00 95.17  moveto (A) center show 
0.00 124.33  moveto (C) center show 
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 0.00 153.50  moveto (C) center show grestore
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 0.00 182.67  moveto (G) center show grestore
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 0.00 211.83  moveto (A) center show grestore
0.005 setlinewidth 1 setlinejoin 1 setlinecap
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 70.00 221.83 moveto (5') center show grestore
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 70.00 -14.00 moveto (3') center show grestore
Il 70.00 215.83 70.00 186.67 LINE
 Il 70.00 186.67 70.00 157.50 LINE
 Il 70.00 157.50 70.00 128.33 LINE
 Il 70.00 128.33 70.00 99.17 LINE
 Il 70.00 99.17 70.00 70.00 LINE
 Il 70.00 70.00 70.00 52.50 LINE
 Il 70.00 52.50 70.00 35.00 LINE
 Il 70.00 35.00 70.00 17.50 LINE
 Il 70.00 17.50 70.00 0.00 LINE
 /Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al -0.00 -14.00 moveto (5') center show grestore
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 0.00 221.83 moveto (3') center show grestore
Tl 0.00 0.00 0.00 17.50 LINE
 Tl 0.00 17.50 0.00 35.00 LINE
 Tl 0.00 35.00 0.00 52.50 LINE
 Tl 0.00 52.50 0.00 70.00 LINE
 Tl 0.00 70.00 0.00 99.17 LINE
 Tl 0.00 99.17 0.00 128.33 LINE
 Tl 0.00 128.33 0.00 157.50 LINE
 Tl 0.00 157.50 0.00 186.67 LINE
 Tl 0.00 186.67 0.00 215.83 LINE
 /Times findfont 8 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Tl -11.00 211.83 moveto (10) center show grestore
gsave Tl 79.00 95.17 moveto (5) center show grestore
gsave Tl -11.00 66.00 moveto (5) center show grestore
gsave Tl 72.91 -15.64 moveto (10) center show grestore
Al W2 70.00 58.50 70.00 64.00 gsave  DASHLINE grestore
 NP W1
 Xl 62.00 215.83 8.00 215.83 LINE
 NP W1
 Xl 62.00 184.67 8.00 184.67 LINE
 Xl 62.00 188.67 8.00 188.67 LINE
 NP W1
 Xl 62.00 155.50 8.00 155.50 LINE
 Xl 62.00 159.50 8.00 159.50 LINE
 NP W1
 Xl 62.00 126.33 8.00 126.33 LINE
 Xl 62.00 130.33 8.00 130.33 LINE
 NP W1
 Xl 62.00 99.17 8.00 99.17 LINE
 NP W1
 Xl 62.00 70.00 8.00 70.00 LINE
 
 showpage 
