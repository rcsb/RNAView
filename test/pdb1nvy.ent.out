PDB data file name: pdb1nvy.ent_new
-----------------------------------------------------------
CRITERIA USED TO GENERATE BASE-PAIR: 
  3.40 --> upper H-bond length limits (ON..ON).
 26.00 --> max. distance between paired base origins.
  2.50 --> max. vertical distance between paired base origins.
 65.00 --> max. angle between paired bases [0-90].
  5.40 --> min. distance between RN9/YN1 atoms.
  8.00 --> max. distance criterion for helix break[0-12]
-----------------------------------------------------------
BASE-PAIR INSTRUCTIONS: 
Column 1 is rnaview assigned base numbers n1_n2, start from 1.
Column 2 & 3 are chain ID & residue number in input PDB file.
Column 4 is for base pair. The left & right are the bases as 
         identified by column 2 & 3 and 5 & 6.
Column 5 & 6 are residue number & chain ID in input PDB file.
Column 7 is for base pair annotation. The standard Watson-Crick
         (W.C.) pairs are annotated as -/- (AU,AT) or +/+ (GC).
         Other pairs are annotated as Leontis_Westhof Classification.
         The three edges: W(Watson-Crick); H(Hoogsteen); S(suger).
         e.g. W/H means the pair is edge of Watson-Crick & Hoogsteen.
Column 8 is glycosidic bond orientation (either cis or trans).
         e.g. 'W/H cis' means the pair is interaction on Watson-Crick
         and Hoogsteen side, glycosidic bond orientation is 'cis'.
Column 9 corresponds to Saenger Classification.

Other columns: 
        Syn sugar-base conformations are annotated as (syn).
        Stacked base pairs are annotated as (stack).
        Non-identified edges are annotated as (.) or (?)
        Tertiary interactions are marked by (!) in the line.
Reference:
Yang et al (2003) Nucleic Acids Research, Vol31,No13,p3450-3461.
-----------------------------------------------------------
BEGIN_base-pair
     1_20, A:     1 T-A    10 B: -/- cis      syn  XX
     2_19, A:     2 C-G     9 B: +/+ cis    syn syn  XIX
     3_18, A:     3 G-C     8 B: +/+ cis    syn syn  XIX
     4_17, A:     4 G-C     7 B: +/+ cis    syn    XIX
     5_16, A:     5 T-A     6 B: -/- cis         XX
     6_15, A:     6 A-T     5 B: -/- cis         XX
      6_7, A:     6 A-C     7 A: S/H cis         !(s_s)
END_base-pair
  The total base pairs =   6 (from   20 bases)
------------------------------------------------
 Standard  WW--cis  WW-tran  HH--cis  HH-tran  SS--cis  SS-tran
        6        0        0        0        0        0        0
  WH--cis  WH-tran  WS--cis  WS-tran  HS--cis  HS-tran
        0        0        0        0        0        0
------------------------------------------------
